### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AkiaCode**
## Status

- [ ] These changes have been tested against Discord API or do not contain API change.
- [x] This PR includes only documentation changes, no code change.
- [ ] This PR introduces some Breaking changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,22 +21,22 @@
 
 You can import the package from https://deno.land/x/harmony/mod.ts (with latest version) or can add a version too, and raw GitHub URL (latest unpublished version) https://raw.githubusercontent.com/harmonyland/harmony/main/mod.ts too.
 
-We also have a (fancy) custom registry for importing Harmony! It's at [code.harmony.rocks](https://code.harmony.rocks), example import URL: `https://code.harmony.rocks/v2.6.1`.
+We also have a (fancy) custom registry for importing Harmony! It's at [code.harmony.rocks](https://code.harmony.rocks), example import URL: `https://code.harmony.rocks`.
```
Comment (Helloyunho) [inline README.md:24]: remove this as harmony.rocks is unavailable now

Comment (AkiaCode) [reply README.md:24]: what?

Comment (Blocksnmore) [reply README.md]: > what?

The domain expired (it now redirects to a namecheap landing page)

**Review by Helloyunho — APPROVED** [review]
lgtm

**Pull request merged**